ci/python-publish.yml

```diff
@@ -12,28 +12,42 @@ on:
   release:
     types: [published]
 
-permissions:
```

Comment:
It might be good to keep this globally in case people start modifying the jobs post workflow creation.

Reply:
Makes sense, I'll re-add!